demos/m5/bootstrap.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+COSIGN_SECRET_NAME="cosign-public-key"
+NAMESPACE="managed"
+QUAY_ROBOT_ACCOUNT="hacbs-release-tests+m5_robot_account"
```

Comment:
Is this a typo or intended? The `+` looks weird

Reply:
Intended.